nvidia-gpu/nvidia-container-toolkit/nvidia-container-cli/pkg.yaml

```diff
@@ -62,6 +62,9 @@ steps:
 
         patch -p0 < /pkg/patches/libnvidia-container/Makefile.patch
 
+        # workaround for Debian 11 (Bullseye): https://github.com/NVIDIA/libnvidia-container/issues/116#issuecomment-826146121
+        make || true
+        mv deps/src/elftoolchain-0.7.1/libelf/'name libelf.so.1' deps/src/elftoolchain-0.7.1/libelf/libelf.so.1
```

Comment:
i've seen this potential fix, maybe we could build libelf first, then build the actual one :thinking: